bb_wrapper/wrapper/cobrancas.py

```diff
@@ -16,11 +34,49 @@ def registra_boleto(self, data):
         response = self._post(url, data)
         return response
 
-    def consulta_boleto(self, numero):
+    def consulta_boleto(self, our_number):
         """"""
+
+        assert len(our_number) == 20, "O nósso número não tem 20 dígitos!"
+
         self.authenticate()
         url = self._construct_url(
-            identifier=numero, search={"numeroConvenio": self._convenio_number}
+            identifier=our_number, search={"numeroConvenio": self.__convenio}
         )
         response = self._get(url)
         return response
+
+    def build_our_number(self, number):
+        """
+        Método para construir o 'Nosso Número'.
+
+        20 dígitos, que deverá ser formatado da seguinte forma:
+            “000” + (número do convênio com 7 dígitos) + (10 algarismos)
```

Comment:
Essas aspas duplas eram de outro tipo rsrs

```python 
>>> ord('“')
8220
>>> ord('"')
34
```

```suggestion
            '000' + (número do convênio com 7 dígitos) + (10 algarismos)
```